### DISCUSSION TITLE
Error: UnsupportedRestoreImageError() When using tart create --from-ipsw

### DISCUSSION THREAD

**Discussion by rant-ci-agent** [cirruslabs/tart#536] [Q&A]
I'm trying to create a Vm using `tart create runner --disk-size=80 --from-ipsw=<ipsw-url>` and after waiting for a long time for it to download the 12.5GB IPSW, it displays:
> Error: UnsupportedRestoreImageError()

I tried again incase it was a one-off, and the second time it took the same amount of time to download the 12.5GB of IPSW and then displayed:
> Using cached *.ipsw file...
> Error: UnsupportedRestoreImageError()

(If it was using a cached IPSW, why did it download 12.5GB again?)

The IPSW I was trying to use is the latest Mac universal 13.4.1 for Mac Studio M2 Max (the host machine I'm using): https://ipsw.me/download/Mac14,13/22F2083

Why is it unsupported? Can I do anything to resolve it?

**Comment by fkorotkov** [comment]
What's the version of macOS on your host? To run and create VMs with macOS 13.3+ you need a host with 13.3+.

**Reply by rant-ci-agent** [reply]
The host is running 13.4.1, exact same build as the one I'm downloading.
<img width="392" alt="Screenshot 2023-07-03 at 17 07 25" src="https://github.com/cirruslabs/tart/assets/138502550/15f72707-79e8-4034-ab0d-ce7b212e656a">

**Comment by edigaryev** [comment]
Could you try the `22F82` image for `13.4.1` instead? It can be obtained [here](https://ipsw.me/download/Mac14,3/22F82).

I'm not sure what's the difference between them, but the `22F2083` one doesn't work for me either, while the `22F82` works just fine.
